app/Transformers/ScoreTransformer.php

```diff
@@ -61,38 +58,19 @@ class ScoreTransformer extends TransformerAbstract
         'current_user_attributes',
     ];
 
-    private string $transformFunction;
+    private bool $legacyFormat;
 
-    public static function newSolo(): static
```

Comment:
`PlaylistItemTransformer` still has a reference to this

Reply:
how did I miss it ಠ_ಠ